src/components/TrackContributing.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export function TrackContributing({
+  trackId,
+}: {
```

Comment:
For consistency, you might want to extract this to `TrackContributingProps`

Reply:
Will do!